src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlArrayTranslator.cs

```diff
@@ -33,6 +33,12 @@ public class NpgsqlArrayTranslator : IMethodCallTranslator, IMemberTranslator
             typeof(Enumerable).GetTypeInfo().GetMethods(BindingFlags.Public | BindingFlags.Static | BindingFlags.DeclaredOnly)
                 .Single(mi => mi.Name == nameof(Enumerable.Any) && mi.GetParameters().Length == 1);
 
+        [NotNull] static readonly MethodInfo StringArrayJoin =
```

Comment:
Uh, EnumerableExtensions.Join is an internal extension method we use inside EFCore.PG, did you mean to translate this?